.pre-commit-config.yaml

```diff
@@ -1,11 +1,11 @@
-###########################################################
-# ruff on both ./code/* and ./core/ during infra prep phase
-###########################################################
-
-files: "code\/|core\/"
+files: "core\/"
```

Comment:
I believe that this should be `"core\/|notebooks\/"`

Reply:
Good catch. I will update in subsequent PR